### PR TITLE
Add troubleshoot help to install NuGet packages

### LINKED ISSUE #19541: Add help to install NuGet packages from a different repository
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-nuget-registry

### What part(s) of the article would you like to see updated?

The "Troubleshooting" needs an additional bulletpoint, explaining what to do to install a NuGet package from a different repository. This is important in my view since installing a package from a different repository is more common that publishing and installing the NuGet package from the same repository.

The way to solve this is by using a personal access token (PAT) in the `dotnet nuget add source` command. This would also clarify that using a PAT for authentication within a GitHub action is perfectly fine (the current structure of the doc indicates that using a PAT is only possible within a *nuget.config* file.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by rafaelschlatter**
### Why:
Closes #19541 
The documentation on how to install NuGet packages from GitHub packages is missing an important point. If installing a NuGet package from a different repository, authentication using a `GITHUB_TOKEN` in the GitHub action will fail. A personal access token (PAT) needs to be used instead. Closes issue #19541 .

### What's being changed (if available, include any code snippets, screenshots, or gifs):

Added an additional bulletpoint in the "Troubleshooting" section. It explains how to authenticate within a GitHub action when installing a NuGet package from a different repository.

### Check off the following:

- [x] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by cmwilson21** [comment]
@rafaelschlatter Thanks for opening an issue and a PR! ✨ I edited your post slightly just so it connects to your issue. 

I'll get this triaged for review! ⚡

**Comment by vgrl** [comment]
Thanks for this contribution @rafaelschlatter! ⚡  Earlier in the same article, in the "Authenticating to GitHub Packages" section, we already mention the limitation of using the `GITHUB_TOKEN`, however I agree it may assist other users to also mention this in the "Troubleshooting" section towards the end of the article.

As such I'm going to make a few changes to your proposed language, and then I'll merge this.

File: content/packages/working-with-a-github-packages-registry/working-with-the-nuget-registry.md [hunk 1]
```diff
@@ -232,6 +232,8 @@ Your NuGet package may fail to push if the `RepositoryUrl` in *.csproj* is not s
 
 If you're using a nuspec file, ensure that it has a `repository` element with the required `type` and `url` attributes.
 
+Using a `GITHUB_TOKEN` for authentication in a GitHub action to install a NuGet package from another repository will fail. The `GITHUB_TOKEN` is repository-scoped. Use a personal access token (PAT) instead.
```
Comment (vgrl) [inline content/packages/working-with-a-github-packages-registry/working-with-the-nuget-registry.md:235]:
```suggestion
If you're using a `GITHUB_TOKEN` to authenticate to a {% data variables.product.prodname_registry %} registry within a {% data variables.product.prodname_actions %} workflow, the token cannot access private repository-based packages in a different repository other than where the workflow is running in. To access packages associated with other repositories, instead generate a PAT with the `read:packages scope` and pass this token in as a secret.
```

**Review by vgrl — COMMENTED** [review]
For clarity, let's expand on the situation that may result in the problem, and then say what the solution is.

**Review by vgrl — APPROVED** [review]
👍 Checked on [preview](https://docs-19542-6b3f5f.preview.ghdocs.com/en/packages/working-with-a-github-packages-registry/working-with-the-nuget-registry#troubleshooting) and all looks good! I'll go ahead and merge this now ⚡

**Pull request merged**